app/src/main/java/ca/chronofit/chrono/onboarding/OnBoardActivity.kt

```diff
@@ -76,10 +77,27 @@ class OnBoardActivity : BaseActivity() {
         })
 
         bind.next.setOnClickListener { nextPage() }
-        bind.skip.setOnClickListener { skip() }
-        bind.continueButton.setOnClickListener { launchMain() }
+        bind.skip.setOnClickListener {
+            val props = JSONObject()
+            props.put("source", "OnBoardActivity")
+            props.put("skipped_at_page", bind.pager.currentItem)
+
+            mixpanelAPI.track("On-boarding skipped", props)
+            skip()
+        }
+        bind.continueButton.setOnClickListener {
+            val props = JSONObject()
+            props.put("source", "OnBoardActivity")
+            mixpanelAPI.track("Finished On-boarding", props)
```

Comment:
"on-boarding" instead of "On-boarding" for the sake of consistency (all the other events are like that)?